doc/api/net.md

```diff
@@ -1006,6 +1006,10 @@ Set the encoding for the socket as a [Readable Stream][]. See
 ### `socket.setKeepAlive([enable][, initialDelay])`
 <!-- YAML
 added: v0.1.92
+changes:
+  - version: v12.17.0
```

Comment:
```suggestion
  - version:
    - v13.12.0
    - v12.17.0
```

Reply:
thanks @aduh95.
can I apply your commit suggestion via the GH UI or is it better to update the original commit?

Reply:
Do what's more convenient to you :) On landing, our tooling will squash the commits into one, so GH UI is perfectly suitable.